tests/histogram.rs

```diff
@@ -1,7 +1,5 @@
 //! Tests from HistogramTest.java
 
-#![allow(non_snake_case)]
```

Comment:
I'd prefer these in a separate PR. I know it's a little annoying, but makes things cleaner.